public/js/controllers/time_controller.js

```diff
@@ -61,7 +61,7 @@ export default class extends Controller {
       if (isCorrectVal(el.dataset.age)) {
         el.textContent = humanize.timeSince(el.dataset.age)
       } else if (el.dataset.age !== '') {
-        el.textContent = humanize.timeSince(Date.parse(el.dataset.age) / 1000)
+        el.textContent = humanize.timeSince((Date.parse(el.dataset.age) / 1000) - (new Date().getTimezoneOffset() * 60))
```

Comment:
I think part of the issue at this point is that the way the dcrdata is written it assumes you have your server set to GMT, but you have yours in your local timezone.  This goes back to my point about TimeDef not having a timezone.  Some of this can be resolved later by making changes to the way time is handled in Go.  Maybe the backend should make sure all times are returned in GMT instead of whatever the server is set to.

Reply:
Sure, that's another way to simplify things. I think most servers are by default set to UTC so  this would mainly occur in local dev. I think simply sending along a unix stamp from the backend would also simplify things and remove the need for loading moment to do conversions.

Reply:
If we just do a better job of making times returned in a consistent format then we probably don't need momentjs.  I'm changing my timezone to local time to make sure that's the issue buck54321 is seeing.

Reply:
So, do we all agree then to send along unix timestamps from backend everywhere we do age calcs?

The alternative of using the current backend data ( date + locale ) and computing offsets on the client is definitely more complex.

Reply:
I suppose your original suggestion of simply expecting the backend to be in UTC is the simplest option of all. 
I think most remote servers have that set as the default. Maybe just add a note in the docs about it and call it done!

Reply:
For now just adding a note to the docs would be fastest.  Updating the backend to return a unix timestamp (in UTC) would make it more robust.  I'm open to suggestions.  You guys are in charge, I'm new.